diff --git a/drivers/gpio/gpiolib.c b/drivers/gpio/gpiolib.c
index 53ff25a..ae77e8d 100644
--- a/drivers/gpio/gpiolib.c
+++ b/drivers/gpio/gpiolib.c
@@ -2852,6 +2852,10 @@ static struct gpio_desc *of_find_gpio(struct device *dev, const char *con_id,
 	return desc;
 }
 
+static bool acpi_lookup_can_try_crs = false;
+module_param(acpi_lookup_can_try_crs, bool, 0644);
+MODULE_PARM_DESC(acpi_lookup_can_try_crs, "Allow ACPI lookups to fall back to _CRS. Required for Bay Trail byt-max98090 audio driver.");
+
 static struct gpio_desc *acpi_find_gpio(struct device *dev,
 					const char *con_id,
 					unsigned int idx,
@@ -2881,7 +2885,7 @@ static struct gpio_desc *acpi_find_gpio(struct device *dev,
 
 	/* Then from plain _CRS GPIOs */
 	if (IS_ERR(desc)) {
-		if (!acpi_can_fallback_to_crs(adev, con_id))
+		if (!acpi_lookup_can_try_crs && !acpi_can_fallback_to_crs(adev, con_id))
 			return ERR_PTR(-ENOENT);
 
 		desc = acpi_get_gpiod_by_index(adev, NULL, idx, &info);
